test/extended/testdata/cmd/test/cmd/deployments.sh

```diff
@@ -81,7 +81,7 @@ os::cmd::expect_success 'oc delete all --all'
 sleep 1
 os::cmd::expect_success 'oc delete all --all'
 
-os::cmd::expect_success 'oc process -f ${TEST_DATA}/application-template-dockerbuild.json -l app=dockerbuild | oc create -f -'
+os::cmd::expect_success 'oc process -f ${TEST_DATA}/application-template-dockerbuild-dc.json -l app=dockerbuild | oc create -f -'
```

Comment:
Is this test the only existence reason of application-template-dockerbuild-dc.json?. If yes, why don't we remove this test as well as this json file?

Reply:
As far as I understand, this is the only piece in origin referencing deploymentconfig and it is better to remove it.

Reply:
deployment.sh is still testing some of the DC functionality. I'd prefer to keep it and remove it once deployment.sh gets migrated to a go based test.